.travis.yml

```diff
@@ -4,19 +4,39 @@ language: bash
 dist: trusty
 
 env:
+  - VERSION=10.0-jdk11
+  - VERSION=10.0-jdk11-slim
+  - VERSION=10.0-jdk14
+  - VERSION=10.0-jdk14-slim
+  - VERSION=10.0-jre11
+  - VERSION=10.0-jre11-slim
+  - VERSION=11.0-jdk11
+  - VERSION=11.0-jdk11-slim
+  - VERSION=11.0-jdk14
+  - VERSION=11.0-jdk14-slim
+  - VERSION=11.0-jre11
+  - VERSION=11.0-jre11-slim
   - VERSION=9.2-jre8
   - VERSION=9.3-jre8
-  - VERSION=9.4-jre8
+  - VERSION=9.4-jdk11
+  - VERSION=9.4-jdk11-slim
+  - VERSION=9.4-jdk14
+  - VERSION=9.4-jdk14-slim
+  - VERSION=9.4-jdk8
+  - VERSION=9.4-jdk8-slim
   - VERSION=9.4-jre11
+  - VERSION=9.4-jre11-slim
+  - VERSION=9.4-jre8
+  - VERSION=9.4-jre8-slim
```

Comment:
Can this list be generated when an update.sh is done?

Reply:
It would be even better if we could do this with some commands regex or bash commands in `.travis.yml`. I tried but couldn't figure out how to do that, I've made the change so that `update.sh` updates this for now.